Podfile

```diff
@@ -323,8 +323,8 @@ end
 ## ===================
 ##
 def wordpress_mocks
-  pod 'WordPressMocks', '~> 0.0.6'
-  # pod 'WordPressMocks', :git => 'https://github.com/wordpress-mobile/WordPressMocks.git', :commit => ''
+  # pod 'WordPressMocks', '~> 0.0.6'
+  pod 'WordPressMocks', :git => 'https://github.com/wordpress-mobile/WordPressMocks.git', :branch => 'add/screenshot-mocks'
```

Comment:
We'll need to merge [this PR ](https://github.com/wordpress-mobile/WordPressMocks/pull/14)first!